webapp/platform/components/src/compass/components/index.ts

```diff
@@ -4,10 +4,12 @@
 import Typography from './typography/typography';
 import Button, {ButtonProps} from './button/button';
 import IconButton from './icon_button/icon_button';
+import GenericModal from './modals/generic_modal';
 
 export {
     IconButton,
     Button,
+    GenericModal,
```

Comment:
is there a reason we still call this GenericModal instead of Modal, i am just following the other naming patterns you have used here like Button, IconButton

Reply:
Yeah we have base modal as well not exported yet, but it will be used for some specific cases we have in the webapp.